examples/website/collision-filter/README.md

```diff
@@ -0,0 +1,27 @@
+This is a standalone version of the CollideExtension example on [deck.gl](http://deck.gl) website.
+
+### Usage
+
+Copy the content of this folder to your project. 
+
+```bash
+# install dependencies
+npm install
+# or
+yarn
+# bundle and serve the app with vite
+npm start
+```
+
+### Data sources
+
+#### Road accidents
```

Comment:
Data credit needs to be updated

Reply:
@Pessimistress `WIP` - I'll inform you about the following review.

Reply:
Data credits updated